Constructive Code Review/outline.txt

```diff
@@ -0,0 +1,205 @@
+- Splash screen
+    - PR screenshot: “Your code is bad and you are bad. Have a bad day.”
+- Importance
+    - Who's done code review before?
+        - Who's done it sync?
+            - We don't really do that, aside from maybe some pair
+              programming.
```

Comment:
I have noticed that sometimes it really helps to hop on a video or voice call to walk through something especially tricky. More async communications like IRC/mailing list can be hard when people have lots of little questions that they might be embarrassed asking "out loud" somewhere archived.